### DISCUSSION TITLE
eslint significantly slower in my dev environment compared to ci

### DISCUSSION THREAD

**Discussion by MichaelKim0407** [eslint/eslint#17502] [Q&A] [answered]
Locally I run everything inside a `node:18` docker container, and the whole project folder is mounted into the container as a volume. I use `yarn` for the project. My `lint` script is `eslint --ext .js --ext .jsx src`.
When I run `yarn lint` it takes ~60s before it tells me OK.

I have the same command running in BitBucket CI, and the `yarn lint` command only takes ~5s. The BitBucket CI script is pretty straightforward:
```
- yarn install --frozen-lockfile
- yarn lint
```

And in theory my machine should be more powerful than the BitBucket CI VM? So I really don't know why it's so much slower on my local machine. I even tried pushing an error just to confirm that the CI is actually working.

Any ideas? Thanks!

**Comment by Rec0iL99** [comment]
Hi @MichaelKim0407, can you please provide a reproducible repo?

**Reply by MichaelKim0407** [reply, marked as answer]
Hi, after doing some research it seems that it's a problem with running `yarn` on Docker for Windows, so it's not specific to eslint. It's just that I always lint before commit so it's most noticeable for me.

Some relevant discussion here: https://github.com/yarnpkg/yarn/issues/7747

Thanks for your help, though!
